tests/sav/syntax.res

```diff
@@ -0,0 +1 @@
+examples/syntax.nit:46,2--15: Error: dev package for `c` unknown by `pkg-config`, install it with `apt-get`, `brew` or similar.
```

Comment:
It is intended (`libc` is never handled by `pkg-config`) so the compiler won’t try to actually build a module that makes no sense (embeds multiple languages, has a lot of possibly-conflicting annotations…). However, because we get a semantic error, we know that it is at least syntactically correct.